modules/core/example/ts/avaxc-test.ts

```diff
@@ -0,0 +1,45 @@
+import { BitGo } from 'bitgo';
+const bitgo = new BitGo({
+  env: 'test',
+});
+const coin = 'tavaxc';
+const walletLabel = 'Test TAVAXC V2 Wallet - ' + (new Date()).getTime();
+const accessToken = '';
+const walletId = '';
+const walletPassphrase = '';
+
+async function createWallet() {
```

Comment:
Can you split this test into `avaxc/create-wallet.ts` and `avaxc/send-hop-transaction`?
That way we can prevent this file from getting too big and it's easier to find the example you need too.